src/main/java/xyz/groundx/caver_ext_kas/wallet/KASWallet.java

```diff
@@ -155,6 +202,24 @@ public AccountSummary disableAccount(String address) throws KASAPIException {
 
     /**
      * Sign a transaction instance using an account in KAS Wallet service.
+     *
+     * <pre>Example
+     * {@code
+     * String fromAddress = "0x{fromAddress}";
+     * String toAddress = "0x{toAddress}";
+     *
+     * ValueTransfer tx = new ValueTransfer.Builder()
```

Comment:
Can we use `caver.transaction.valueTransfer.create`?

Reply:
@jimni1222 
Great idea. I'll apply this feedback to this PR soon.